drivers/metadata/metadata.go

```diff
@@ -466,14 +467,15 @@ func (i IndexSet) Get() *Index {
 }
 
 type Index struct {
-	Catalog   string
-	Schema    string
-	Table     string
-	Name      string
-	IsPrimary Bool
-	IsUnique  Bool
-	Type      string
-	Columns   string
+	Catalog      string
+	Schema       string
+	Table        string
+	Name         string
+	IsPrimary    Bool
+	IsUnique     Bool
+	Type         string
+	Columns      string
+	AccessMethod string
```

Comment:
Is there a case when both index type and access method are used at the same time? If not, maybe we shouldn't add a new field here and only return access method as the type in the postgresql driver? The query that fetched this should use COALESCE

Reply:
I added suggested changes.